packages/forms/frontend/sirius-components-widget-reference/src/modals/BrowseModal.types.ts

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { GQLReferenceWidget } from '../ReferenceWidgetFragment.types';
+
+export interface BrowseModalProps {
+  editingContextId: string;
+  widget: GQLReferenceWidget;
+  onClose: (selectedElementIds: Array<string>) => void;
```

Comment:
Sorry I didn't see it before. I think it should be `string[]` instead of `Array<string>`, to be consistent with ou code base.

Reply:
Done